### ISSUE TITLE
Ping should return valid JSON for 200 status code

### ISSUE THREAD

**Issue by cancan101** [4Catalyzer/flask-resty#174]
Given we return `200` status code rather than `204` (no content), we should return valid json. eg. `{}` or `{'status': 'ok'}` Maybe take the return values as kwarg and default to empty dict?
https://github.com/4Catalyzer/flask-resty/blob/4c167f319c0ce0d4745968d8c69d333e7220504f/flask_resty/api.py#L154-L169

**Comment by itajaja** [comment]
true, but I'd rather return 204 then, unless there is a specific reason for returning a payload?

**Comment by cancan101** [comment]
204 seems fine but 1. Confirm elb now allows specifying return code (I believe it does). 2. Be aware that this will be an unexpected breaking  change for anyone using it and expecting a 200 (eg current elb users that haven't set 204 as acceptable).

**Comment by itajaja** [comment]
fair points :)
